commands/raids/set-time.js

```diff
@@ -30,13 +30,12 @@ class SetTimeCommand extends Commando.Command {
 			return;
 		}
 
-		const start_time = raid.args[0];
-
-		const info = Raid.setRaidTime(message.channel, message.member, raid.raid.id, start_time);
-		let total_attendees = Raid.getAttendeeCount({raid: info.raid});
+		const start_time = raid.args[0],
+			info = Raid.setRaidTime(message.channel, message.member, raid.raid.id, start_time),
+			total_attendees = Raid.getAttendeeCount({raid: info.raid});
 
 		for (let i = 0; i < info.raid.attendees.length; i++) {
-			let member = info.raid.attendees[i];
+			const member = info.raid.attendees[i];
```

Comment:
Interesting, I didn't realize that const was actually acting as a "let" such that the scope was restricted.  I thought for sure this would throw an error b/c it's trying to constantly change a const variable, but instead of it being something that could be accessed outside of this function, it's actually fine b/c each loop is creating another variable.  Very interesting! :P